db_stress_tool/db_stress_tool.cc

```diff
@@ -286,6 +286,14 @@ int db_stress_tool(int argc, char** argv) {
     exit(1);
   }
 
+  if (FLAGS_manual_wal_flush_one_in > 0 && !FLAGS_sync_fault_injection) {
```

Comment:
I'd actually change the condition to enable the tracing when manual_wal_flush_one_in > 0 rather than require it to run with sync_fault_injection=1 to enable tracing: https://github.com/facebook/rocksdb/blob/c76a90ceb97195135397c1e35bba4f24353b5be1/db_stress_tool/db_stress_test_base.cc#L328

That way we can verify that `FlushWAL(false /* sync */)` on a DB with `manual_wal_flush == true` makes that DB prefix recoverable in case of a process crash.

Reply:
Interesting - I will take a look. Also this reminds me of the left-out case where disable_wal = 0 -> manual_wal_flush = 0

Reply:
Fixed